doc/credentials.md

```diff
@@ -0,0 +1,230 @@
+# Credentials
+Camera credentials are stored in the EdgeX Secret Store and referenced by MAC Address. All 
+devices by default are configured with credentials from `DefaultSecretPath` unless configured
+as part of a group within `AppCustom.CredentialsMap`.
+
+Three things must be done in order to add an authenticated camera to EdgeX:
+- Add device to EdgeX
+  - Manually or via auto-discovery
+- Add `Credentials` to `Secret Store`
+  - Manually or via utility scripts
+- Map `Credentials` to devices
+  - Manually or via utility scripts
+  - Configure as `DefaultSecretPath`
+
+## Terminology / Definitions
+- **Secret**: A generic object which is stored as key/value pairs in the `Secret Store` under a specific `Secret Path` key.
```

Comment:
```suggestion
- **Secret**: A generic object which is stored as a key/value pair in the `Secret Store` under a specific `Secret Path` key.
```

Reply:
The wording can be improved, but it is actually multiple k/v pairs not just one. Its basically a map.